src/integration/integration_scheduler_scalingengine_test.go

```diff
@@ -96,7 +96,7 @@ var _ = Describe("Integration_Scheduler_ScalingEngine", func() {
 
 			Eventually(func() bool {
 				return activeScheduleExists(testAppId)
-			}, 2*time.Minute).Should(BeFalse())
+			}, 2*time.Minute, 5*time.Second).Should(BeFalse())
```

Comment:
I see, is it worrying for real usage that the difference between polling every 5 seconds vs every second can make such a big difference? I guess what I'm asking is if polling every 1 second causes enough cpu load to affect other tests that seems like it could indicate an actual performance problem?